week2/src/backend/index.js

```diff
@@ -0,0 +1,120 @@
+
+const express = require("express");
+const app = express();
+const  mealsRouter = require("./routes/meals");
+const  reservationsRouter = require("./routes/reservations");
+const  reviewsRouter = require("./routes/reviews");
+
+
+app.use("/meals",function(req,res,next){
+  const meals =req.query;
+  const maxPrice = req.query.maxPrice;
+  const title=req.query.title ;
+  const createdAfter =req.query.createdAfter;
+  const limit =req.query.limit;
+  // Check if the object is empty
+  function isEmpty(obj) {
```

Comment:
Put a function like this into a seperate file maybe called utils.js. Here it is just a little messy